src/usePopper.js

```diff
@@ -68,14 +68,14 @@ export const usePopper = (
       },
       requires: ['computeStyles'],
     }),
-    [setState]
```

Comment:
While `setState` does change on each render, it actually doesn't invalidate the cache! As it's bound to the same context, and React is smart enough somehow.
You can test this with the following:
```js
useMemo(() => {
  console.log('changed?');
}, [setState]);
```